installing/installing_alibaba/installing-alibaba-vpc.adoc

```diff
@@ -0,0 +1,71 @@
+:_content-type: ASSEMBLY
+[id="installing-alibaba-vpc"]
+= Installing a cluster on Alibaba Cloud into an existing VPC
+include::_attributes/common-attributes.adoc[]
+:context: installing-alibaba-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster into an existing Alibaba Virtual Private Cloud (VPC) on Alibaba Cloud Services. The installation program provisions the required infrastructure which can then be customized. To customize the VPC installation, modify parameters in the 'install-config.yaml' file before you install the cluster.
```

Comment:
```suggestion
In {product-title} version {product-version}, you can install a cluster into an existing Alibaba Virtual Private Cloud (VPC) on Alibaba Cloud Services. The installation program provisions the required infrastructure, which can then be customized. To customize the VPC installation, modify the parameters in the 'install-config.yaml' file before you install the cluster.
```

Reply:
I had to re-do this edit in the restored files of my new PR. Thank you so much for your detailed review, @Amrita42 !